docs/getting_started.md

```diff
@@ -69,7 +63,7 @@ We now need to know more about the configuration process, as this is a major par
 
 Shadow requires **XML input files** to configure an experiment. These files are used to describe the structure of the network topology, the network hosts that should be started, and application configuration options for each host. The network, node, and application configuration is specified in the `shadow.config.xml` file; the client behavior models (traffic generator configurations) are specified in the `tgen.*.graphml.xml` files.
 
-Lets take another look at the `tgen` example from above, the configuration for which can be found in the `resource/examples/shadow.config.xml` file. After parsing this file, Shadow creates the internal representation of the network, loads the plug-ins, and generates the virtual hosts. You should examine these configuration files and understand how they are used. For example, you might try changing the quantity of clients, or the bandwidth of the network vertices or the latency of the network edges to see how download times are affected.
+Lets take another look at the `tgen` example from above, the configuration for which can be found in the `resource/examples/shadow.config.xml` file. After parsing this file, Shadow creates the internal representation of the network, loads the applications, and generates the virtual hosts. You should examine these configuration files and understand how they are used. For example, you might try changing the quantity of clients, or the bandwidth of the network vertices or the latency of the network edges to see how download times are affected.
 
 The network topology used for the simulation is also configured inside of the `shadow.config.xml` file. In the example above, the network topology was embedded as CDATA inside of the `<topology>` element. This network topology is itself XML in the standard graphml format, and can be stored in a separate file instead of embedding it. You may then modify `shadow.config.xml` to reference the external graphml topology file rather than embedding it with something like `<topology path="~/.shadow/share/topology.graphml.xml" />`.
```

Comment:
graphml -> GML

Reply:
I left this for now since I think we might want to rewrite this entire section.